### PR TITLE
removes the fluff item from moonbase, modifies loot

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Removes the fluff item from moonbase.
Moves the anomaly core into the safe, as that is more valuable than the research paper.
Paper moved outside the vault.
Added the experimental item lockbox to the safe. This one requires head or sec door access, so an explorer will have to bribe a head or officer to keep the contents.

Note: The ruin has no way to open a safe. You need to bring a stethoscope from somewhere else.

Contents 10% of the time are a facehugger, which will leap out and infect the person without a proper helmet.
This is unlikely to cause a breakout, as the person will need to not be taken to medbay during this time, and not have a mask on. Combined with sec access or head access needed, rare. As you can not look inside, not going to exactly be easy to prepare a monkey as bait either.

If not the hugger, we move onto other loot. 25% of the time an experimental KA. 25% of the time AI camera upgrade. 25% of the time ninja stealth.
The remaining 25% of the time grants some various organs. This will require medical or robotics helping you, and some of them are going to raise eyebrows if the officer or head looks in the box first.
One of these is picked. Drone organs are all together
Dark shard katana, same as mining.
Toolset abductor, good tools or rnd.
Esword arm. Better than straight up giving them an esword as they have to get it implanted, and the officer / heads are going to shut you down.
Pulse heart, bio or charging. 
Eyes of the gods, harder to use xray than the bio 8 implant.
Xeno drone organs WITHOUT hivenode. Could have fun with them, but no stunning neurotoxin, and no hearing xenos spawn.

## Why It's Good For The Game

Fluff items in space ruins are bad.

Moving the stronger loot into the safe is good.

Making a lootbox which many crewmembers could use that requires the explorer to work with a head or officer (or bribe them), with a small chance to go wrong is interesting!

## Testing

Confirm map changed correctly

Confirmed lockbox worked correctly.

## Changelog
:cl:
del: Remove fluff item from moon outpost 19
tweak: Moved anomaly core inside moon outpost 19 safe.
tweak: A lockbox with many experimental items can be found inside the outpost 19 safe.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/lockbox.dm [hunk 1]
```diff
@@ -198,3 +198,36 @@
 		locked = FALSE
 		icon_state = icon_broken
 		origin_tech = null //wipe out any origin tech if it's unlocked in any way so you can't double-dip tech levels at R&D.
+
+/obj/item/storage/lockbox/experimental_weapon
+	name = "class A-113 classified lockbox"
```
Comment (Sirryan2002) [inline code/game/objects/items/weapons/storage/lockbox.dm:203]:
```suggestion
	name = "A-113 classified lockbox"
```
I would just cut down on wordiness here, the presence of a numerical identifier implies some sort of classification or product identifier

**Review by Sirryan2002 — APPROVED** [review]

**Comment by kano-dot** [comment]
i never knew eyepro was a fluff item (despite it was literally in type name and i just noticed), sorry. I like the lockbox lootbox idea, also not related but there's an autolathe in engineering wing for explorers to print a stethoscope once the area is powered up.

**Review by Henri215 — APPROVED** [review]
Nice surprise

File: code/game/objects/items/weapons/storage/lockbox.dm [hunk 2]
```diff
@@ -198,3 +198,36 @@
...
+					if(8)
+						new /obj/item/organ/internal/heart/demon/pulse(src)
+					if(9)
+						new /obj/item/organ/internal/eyes/cybernetic/eyesofgod(src)
+					if(10) //Disected drone before the place got wiped. No hivenode.
+						new /obj/item/organ/internal/alien/plasmavessel/drone(src)
+						new /obj/item/organ/internal/alien/acidgland(src)
+						new /obj/item/organ/internal/alien/resinspinner(src)
+		return
+	new /obj/item/clothing/mask/facehugger(src) //Suprise! Storing facehuggers improperly is what lead to this mess.
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/lockbox.dm:233]: I think this is more readable using prob and picks, with special cases for the facehugger and dissected drone
```suggestion
		if(prob(10))
		new /obj/item/clothing/mask/facehugger(src) //Suprise! Storing facehuggers improperly is what lead to this mess.
		return
	var/spawn_type = pick(/obj/item/gun/energy/kinetic_accelerator/experimental, /obj/item/surveillance_upgrade, /obj/item/mod/module/stealth/ninja)
	if(prob(25))
		if(rand(1, 6) == 1) //organ time. I want this to be more balanced in distribution, so organs are under a prob 25
			new /obj/item/organ/internal/alien/plasmavessel/drone(src)  //Disected drone before the place got wiped. No hivenode.
			new /obj/item/organ/internal/alien/acidgland(src)
			new /obj/item/organ/internal/alien/resinspinner(src)
			return
		spawn_type = pick(
			/obj/item/organ/internal/cyberimp/arm/katana,
			/obj/item/organ/internal/cyberimp/arm/toolset_abductor,
			/obj/item/organ/internal/cyberimp/arm/esword,
			/obj/item/organ/internal/heart/demon/pulse,
			/obj/item/organ/internal/eyes/cybernetic/eyesofgod
		)
	new spawn_type(src)
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/lockbox.dm [hunk 3]
```diff
@@ -198,3 +198,27 @@
...
+			new /obj/item/organ/internal/alien/plasmavessel/drone(src)  //Disected drone before the place got wiped. No hivenode.
+			new /obj/item/organ/internal/alien/acidgland(src)
+			new /obj/item/organ/internal/alien/resinspinner(src)
+			return
+		spawn_type = pick(
+			/obj/item/organ/internal/cyberimp/arm/katana,
+			/obj/item/organ/internal/cyberimp/arm/toolset_abductor,
+			/obj/item/organ/internal/cyberimp/arm/esword,
+			/obj/item/organ/internal/heart/demon/pulse,
+			/obj/item/organ/internal/eyes/cybernetic/eyesofgod)
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/lockbox.dm:223]: This really is just stylistic, but for consistency, we have the ) on the same indentation as the declaration, after the last element.
```suggestion
			/obj/item/organ/internal/eyes/cybernetic/eyesofgod
		)
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/lockbox.dm [hunk 4]
```diff
@@ -215,11 +215,13 @@
...
 			return
-		spawn_type = pick(
+		var/list/organ_loot = list(
 			/obj/item/organ/internal/cyberimp/arm/katana,
 			/obj/item/organ/internal/cyberimp/arm/toolset_abductor,
 			/obj/item/organ/internal/cyberimp/arm/esword,
 			/obj/item/organ/internal/heart/demon/pulse,
 			/obj/item/organ/internal/eyes/cybernetic/eyesofgod
-		)
+			)
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/lockbox.dm:224]:
```suggestion
		)
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Burzah — APPROVED** [review]

**Pull request merged**